Martium.TravelInfo.App/Country ISO/IsoDictionary.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Martium.TravelInfo.App.Country_ISO
+{
+    public class IsoDictionary

```

Comment:
Rename class to `CountryDictionary` and move into `Constants` folder (remove this new folder)